linera-base/src/task.rs

```diff
@@ -138,10 +138,10 @@ mod implementation {
     }
 
     /// The type of a future awaiting another task.
-    pub type NonblockingFuture<R> = oneshot::Receiver<R>;
+    pub type NonBlockingFuture<R> = oneshot::Receiver<R>;
```

Comment:
https://en.wiktionary.org/wiki/nonblocking https://books.google.com/ngrams/graph?content=nonblocking%2Cnon-blocking&year_start=1800&year_end=2022&corpus=en&smoothing=3 :) (I don't really mind though, if we want to make ‘non-blocking’ our standard spelling.)

Reply:
I had to change it because the _other_ (non-`web`) conditionally compiled type was `NonBlockingFuture`. Happy to change the other one instead, if you prefer.